tensorflow/compiler/tf2xla/xla_op_registry.h

```diff
@@ -270,6 +270,8 @@ class XlaOpRegistry {
     // operands and not their values.
     bool is_metadata_op = false;
 
+    string label;
```

Comment:
use std::string please. I know that right now this file uses mostly string instead of std::string, but at some point this will be fixed with a large scale change. But I would prefer if we already start using std::string everywhere when we introduce new variables :)